apps/website/src/pages/jobs/[slug].tsx

```diff
@@ -0,0 +1,154 @@
+            well as developers, and who can transform reused UI patterns such as
+            list items into reusable UI components.
+            <br />
+            <br />
+            The ideal candidate will love finessing UI implementations to the
+            highest levels of quality and will care deeply about things like
+            code cleanliness, reusability, maintainability, performance, and UI
+            layout accuracy - as well as doing whatever needed to create a
+            best-in-class user experience for Status’s users. Willingness to
+            both learn and share your knowledge with others,
+            product-orientation, and making sure all team members are aligned
+            when developing new features, will make you successful in the role.
+            Status is a fast-paced, flat organization, working on cutting edge
+            blockchain and decentralized messaging technologies in a dynamic
+            landscape. We look forward to meeting you!
+          </Text>
+
+          <Text size={19}>
+            The role We’re growing our mobile development team. Join us in
+            building a fully decentralized, censorship resistant, privacy first
+            group chat platform that leverages the Ethereum blockchain to enable
+            individuals and groups worldwide to communicate and transact with
+            one another freely without restriction. Status is looking for an UI
+            Engineer to join our mobile development team who will work closely
+            with Design to transform UI specifications into beautiful, smooth,
+            performant and near pixel perfect interactive interfaces. The ideal
+            person will be comfortable working on features end-to-end, has an
+            eye for design and visual detail, enjoys working with designers as
+            well as developers, and who can transform reused UI patterns such as
+            list items into reusable UI components.
+            <br />
+            <br />
+            The ideal candidate will love finessing UI implementations to the
+            highest levels of quality and will care deeply about things like
+            code cleanliness, reusability, maintainability, performance, and UI
+            layout accuracy - as well as doing whatever needed to create a
+            best-in-class user experience for Status’s users. Willingness to
+            both learn and share your knowledge with others,
+            product-orientation, and making sure all team members are aligned
+            when developing new features, will make you successful in the role.
+            Status is a fast-paced, flat organization, working on cutting edge
+            blockchain and decentralized messaging technologies in a dynamic
+            landscape. We look forward to meeting you!
+          </Text>
+        </div>
+
+        {/* FOOTER */}
+        <div
+          className={cx(
```

Comment:
What is the reason for cx here?

Reply:
better readability – better differentiation between base and `md:*` classes